apps/root/src/services/positionService.ts

```diff
@@ -1500,6 +1544,7 @@ export default class PositionService {
     }
   }
 
+  // TODO: Add price and amounts in usd
```

Comment:
Is this still in TODO? I'm not finding where this is pending

Reply:
Oh shit good catch, actually added this comments so I wouldn't forget, forgot anyway u.u